src/utils/helperTransition/index.tsx

```diff
@@ -3,7 +3,9 @@ import { Transition } from 'react-transition-group';
 
 import { ANIMATION } from '../../constants';
 
-const getStyleForTransitionState = (transitionState: string): object => {
+const getStyleForTransitionState = (
+  transitionState: string,
+): Record<string, unknown> => {
```

Comment:
This is another one where we can remove the type definition of the return value--it infers all of the possible returns:

<img width="434" alt="Screen Shot 2020-10-08 at 8 19 46 AM" src="https://user-images.githubusercontent.com/13544620/95463976-0d8f0b00-093f-11eb-9d38-8b9bb1576305.png">

Reply:
yep removed thanks